xml/security_ldap_replication.xml

```diff
@@ -0,0 +1,645 @@
+<command>--suffix dc=<replaceable>example</replaceable>,dc=<replaceable>com</replaceable></command> \
+<command>--host &exampleserver2ip; --port 3389 --conn-protocol LDAP --bind-dn "cn=replication manager,cn=config"</command> \
+<command>--bind-passwd <replaceable>PASSWORD</replaceable> --bind-method SIMPLE <replaceable>RW2_to_RW1</replaceable></command>
+</screen>
+  <para>
+   Changes made on either RW1 or RW2 will now be replicated to the other. Check
+   replication status on either server with the following command:
+  </para>
+<screen>
+&prompt.sudo;<command>dsconf <replaceable>INSTANCE-NAME</replaceable> repl-agmt status</command> \
+<command>--suffix dc=<replaceable>example</replaceable>,dc=<replaceable>com</replaceable></command> \
+<command>--bind-dn "cn=replication manager,cn=config"</command> \
+<command>--bind-passwd <replaceable>PASSWORD</replaceable> <replaceable>RW2_to_RW1</replaceable></command> 
+</screen>
+</sect3>
+
+ <sect3 xml:id="sec-security-ldap-replication-four-nodes">
+ <title>Configuring a read-only node</title>
+ <para>
+   To create a read-only node, start by creating the replication
+   manager account and metadata. The hostname of the example server
+   is RO3:
+ </para>
+ <screen>&prompt.sudo;<command>dsconf <replaceable>INSTANCE_NAME</replaceable> replication create-manager</command>
+&prompt.sudo;<command> dsconf <replaceable>INSTANCE_NAME</replaceable></command> \
+<command>replication enable --suffix dc=<replaceable>EXAMPLE</replaceable>,dc=<replaceable>COM</replaceable></command> \
+<command>--role consumer --bind-dn "cn=replication manager,cn=config"</command></screen> 
+
+<para>
+  Note that for a read-only replica you do not provide a replica-id, and the 
+  role is set to <literal>consumer</literal>. This allocates a
+  special read-only replica-id for all read-only replicas. After the read-only 
+  replica is created, add the replication agreements from RW1 and RW2 
+  to the read-only instance. The following example is on RW1:
+ </para>
+ <screen>
+&prompt.sudo;<command>dsconf <replaceable>INSTANCE_NAME</replaceable></command> \
+<command>repl-agmt create --suffix dc=<replaceable>EXAMPLE</replaceable>,dc=<replaceable>COM</replaceable></command> \
+<command>--host &exampleserverip; --port 3389 --conn-protocol LDAP</command> \ 
+<command>--bind-dn "cn=replication manager,cn=config" --bind-passwd <replaceable>PASSWORD</replaceable></command> 
+<command>--bind-method SIMPLE <replaceable>RW1_to_RO3</replaceable></command></screen>
+<para>
+  The following example configures the replication agreement between RW2 and 
+  RO1:
+</para>
+<screen>
+&prompt.sudo;<command>dsconf <replaceable>INSTANCE_NAME</replaceable> repl-agmt create</command> \
+<command>--suffix dc=<replaceable>EXAMPLE</replaceable>,dc=<replaceable>COM</replaceable></command> \
+<command>--host &exampleserver2ip; --port 3389 --conn-protocol LDAP</command> \
```

Comment:
Should make it --conn-protocol LDAPS i think .... 

Also --port needs to be a replaceable.

Reply:
:+1: